src/js/components/ServiceForm.js

```diff
@@ -293,7 +293,7 @@ class ServiceForm extends SchemaForm {
       if (!['host', 'bridge'].includes(model.networking.networkType)) {
         // Set expose to true as default
         model.networking.ports.map(function (port) {
-          port.expose = true;
```

Comment:
@Poltergeist currently setting this to false will filter this port mapping from the app definition that we send.